modules/network-observability-quickfilter.adoc

```diff
@@ -7,6 +7,13 @@
 = Filtering the network traffic
 By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the `FlowCollector` instance. You can use the filter options to observe the required data by changing the preset filter.
 
+Query Options::
+You can use *Query Options* to optimize the search results, as listed below:
+
+** *Log Type*: The available options *Conversation* and *Flows* provide the ability to query flows by log type, such as flow log, new conversation, completed conversation, and a heartbeat, which is a periodic record with updates for long conversations. A conversation is an aggregation of flows between the same peers.  
```

Comment:
> available options *Conversation* and *Flows*

I think the options need to be set off somehow. Maybe:
_available options, *Conversation* and *Flows*,_

Reply:
> flow log, new conversation, completed conversation, and a heartbeat, which

I think be adding the modifying _which_ statement, you need semi-colons in the list:
_flow log; new conversation; completed conversation; and a heartbeat, which_

Reply:
Yep, agree.